internal/authctx/helper.go

```diff
@@ -91,10 +91,10 @@ func ProviderConfigureContext(_ context.Context, d *schema.ResourceData) (interf
 	config.TLSConfig.ClientAuthKey, _ = d.Get(clientAuthKey).(string)
 	config.TLSConfig.CaCert, _ = d.Get(caCert).(string)
 
-	return setContext(config)
+	return setContext(&config)
 }
 
-func setContext(config TanzuContext) (TanzuContext, diag.Diagnostics) {
+func setContext(config *TanzuContext) (TanzuContext, diag.Diagnostics) {
```

Comment:
@ishangupta-ds , the proxy changes was manually tested right ?
why this change now?

Reply:
Yeah, the original one should also work as we are just initialising the field here but since for all other functions we have passed as pointer so for consistency we changed here as well.